script/scriptMaster.sh

```diff
@@ -147,7 +147,6 @@ scriptMaster() {
     fi
 
     # get user-selected deploy script and contract from list
-    SCRIPT=$(ls -1 "$DEPLOY_SCRIPT_DIRECTORY" | sed -e 's/\.s.sol$//' | grep 'Deploy' | gum filter --placeholder "Deploy Script")
```

Comment:
this was required to be removed since a similar call was added above and made this an erroneous duplicate